### PR TITLE
Remove the shooting lightning ability from monsters

### PR DESCRIPTION
**Pull request by BalthazarArgall**
#### Summary
Content "Remove the 'Shockstorm' attack from monsters that have it"

#### Purpose of change
I dislike the "Shockstorm" ability for a few reasons but mainly because shooting arcs of electricity at this scale would require absurd amounts of energy. Now, I was told this isn't an issue in itself but I'm still putting the PR up for the sake of discussion and also because it turns out that this belonging in the game may be a "vibe"/"magic" issue that I can't have a say on (not that I'm complaining).

#### Describe the solution
Remove the Shockstorm attack from monsters that had it. The attack stays in the code as well as the monsters because they're probably getting updated in the future.

#### Describe alternatives you've considered
Severely reduce the range of the shockstorm but even a 1m arc would require too much energy for something as big as a hulk to produce, so even then it'd be very very magic.

#### Testing
None, but there is no way this breaks anything (probably).

### CODE REVIEW

**Comment by Swootch** [comment]
I've seen the topic of the shocker zombies come up a few times now. The electric arcs are unrealistic, but it would make sense if the zombies were to spew a stringy slime through which the electric shocks would travel. We already have several variants of spewing zombies, this would fit in with the rest of them.

**Comment by KHeket** [comment]
> it would make sense if the zombies were to spew a stringy slime through which the electric shocks would trave

But in this case shockers should have large capacity of this slime
More realistic, as I think, to make shockers have tentacles, we already have tentacle-hand zombies, or something like long and tine tail/tongue. Tail/tongue touch you - and work similar to long range shockers from real world, also tongue can be slime/wet to improve conductivity

**Comment by aapocalypseAArisen** [comment]
If the topic of  electric energy expenditure is on the table at the moment, then the Incandescent Husks constant cloud of shock is going to be extremely high. 

Also the Zombie Bio-operator is described as having unspecified broken equipment that somehow has enough power to constantly give off lethal amounts of electric energy. I cant fathom what that equipment might even be.

**Comment by anothersimulacrum** [comment]
None is an inappropriate summary line for this. This is exactly the kind of change that belongs in a changelog.

**Comment by BalthazarArgall** [comment]
> None is an inappropriate summary line for this. This is exactly the kind of change that belongs in a changelog.

I have absolutely no idea what the appropriate category for this is. This is not really about balance, it's not really a feature, I'm picking content because it's technically content and I'm removing it I guess.

**Comment by kevingranade** [comment]
This is absolutely the direction we want to go,  I just haven't had the time/ interest to knock it out myself.  I'd *like* to have some kind of stunning and/ or sensory enhancement to replace it,  but that's not necessarily to happen in the same PR.

Affected monsters need to lose their ranged attacker flags,  and it might be cleaner to remove the shocker skeleton entirely since it doesn't seem to have any distinctive abilities left.  Does just the electric flag do enough to justify the whole monster type?
The others have electrical components to their melee attacks and zapback, which is plenty.

**Comment by Procyonae** [comment]
You can wear thick 1mm gloves and zapback will never affect you so I don't see how that's enough

https://github.com/CleverRaven/Cataclysm-DDA/blob/bc9ff31c1555c2c68888080f9e419809e281f692/src/mondefense.cpp#L55-L59

**Comment by BalthazarArgall** [comment]
> This is absolutely the direction we want to go, I just haven't had the time/ interest to knock it out myself. I'd _like_ to have some kind of stunning and/ or sensory enhancement to replace it, but that's not necessarily to happen in the same PR.
> 
> Affected monsters need to lose their ranged attacker flags, and it might be cleaner to remove the shocker skeleton entirely since it doesn't seem to have any distinctive abilities left. Does just the electric flag do enough to justify the whole monster type? The others have electrical components to their melee attacks and zapback, which is plenty.

Alright, cleaned up a little, turns out the zaptor didn't even have the ranged flag to begin with. The shocker skeleton has been removed but I can also bring him back with zapback and electric damage attacks if someone has an intent for it.

**Review by Maleclypse — DISMISSED** [review]
Ok I think this is in the right direction but I'll try and spend some time this weekend tweaking the application.   I would specifically think that a small sparks radius wouldn't be out of the question for these or at least to keep the skeletal shocker with a small sparks radius.  I want to spend some time looking at how often electrical damage zaps players and so I'm marking this up to make sure I remember to do so.


edit: hadn't seen Kevin's comments until just this moment.  I'll defer to those.

**Comment by BalthazarArgall** [comment]
Skeletal shocker has been restored and updated to be more in line with the other "electric" zombies. Are there other changes that need to be made? I can't see what specific changes are being requested.

**Comment by Procyonae** [comment]
In my eyes this PR is removing the only impactful unique quality of the shocker line which is effectively equivalent to removing them.

**Comment by BalthazarArgall** [comment]
> In my eyes this PR is removing the only impactful unique quality of the shocker line which is effectively equivalent to removing them.

Are you suggesting we wait until electric zs get something to "replace" shockstorm before we go through with the removal?

**Comment by ADekema** [comment]
Personally I think that it might be better to remove the skeletal shockers and give the rust/metal zombies the shocking abilities. Bone doesn't conduct electricty very well but metal does. The rust zombies would be a more thematic for the armored shocker niche. You would have to make rust zombies a bit more common and make it so that they don't evolve into other zombie types.

**Comment by kevingranade** [comment]
A taser is not happening.

File: data/json/monsters/zed_skeletal.json [hunk 1]
```diff
@@ -52,6 +52,37 @@
...
+    "melee_damage": [ { "damage_type": "electric", "amount": 12 }, { "damage_type": "cut", "amount": 3 } ],
+    "luminance": 8,
+    "bleed_rate": 0,
+    "harvest": "mr_bones",
+    "special_attacks": [
+      { "id": "grab", "cooldown": 21 },
+      { "id": "scratch_humanoid" },
+      { "id": "bite_humanoid", "cooldown": 5 },
+      [ "PARROT", 8 ],
+      [ "SHOCKSTORM", 25 ]
```
Comment (SirPendrak) [inline data/json/monsters/zed_skeletal.json:75]: So now all electrick zombies lost schockstorm abbility, except for skeletal shocker. He still is undead Zeus.

Comment (BalthazarArgall) [reply data/json/monsters/zed_skeletal.json:75]: Should be fixed by #74693.

**Pull request merged**